index.html

```diff
@@ -127,20 +128,23 @@ <h2 class="about-title">About <br> Myself</h2>
                 developing, a feature that you
                 need built or a project that needs coding. I'll do love to help with it
             </div>
-            <form action="https://formspree.io/f/xvolalez" method="post">
+            <form action="https://formspree.io/f/xvolalez" method="post" id="form">
+                
                 <ul>
-                    <li><label for="name"></label><input type="text" name="for-name" id="name"
-                            placeholder="Put your name" maxlength="30" required></li>
+                    <li><label for="name"></label><input type="text" name="for-name" id="name" placeholder="Put your name"
+                        maxlength="30" required></li>
                     <li><label for="mail"></label><input type="email" name="for-mail" id="mail"
-                            placeholder="put your adress email" required>
-                    </li>
+                        placeholder="put your adress email" required></li>
                     <li><label for="textarea"></label><textarea name="for-text" id="textarea" maxlength="500" cols="30"
-                            rows="10" placeholder="Write your message here" required></textarea></li>
-                    <li><button type="button">Get it Touch</button></li>
-                </ul>
+                        rows="10" placeholder="Write your message here" required></textarea></li>
+                        <li><small id="form-Message">Ooups ! The value entered should be in lowercase</small></li> 
```

Comment:
- [ ] Good job so far 👍 The form cannot be submitted with an uppercase email. It would be great to specify that the value in question here is the email value